graffiti/config.py

```diff
@@ -37,6 +46,8 @@ def __init__(self, cfg, basedir, logdir, precision=2):
         self.name = cfg['NAME']
         self.logdir = logdir
         self.precision = precision
+        self.provider = cfg.get('PROVIDER', None)
```

Comment:
Minor, but `None` is not necessary here.

Reply:
Corrected 8bb1458